web/src/tippyjs.js

```diff
@@ -470,4 +470,13 @@ export function initialize() {
         delay: LONG_HOVER_DELAY,
         appendTo: () => document.body,
     });
+
+    delegate("body", {
+        target: "#groups_overlay .create_user_group_plus_button",
+        content: $t({
+            defaultMessage: "Create new user group",
+        }),
+        placement: "bottom",
+        appendTo: () => document.body,
```

Comment:
Having now seen the commit message, please put a brief version of the explanation in a comment here, since otherwise I bet someone will "simplify" it back to the broken way :)

Reply:
Added a comment to explain these changes.